.releaserc.yml

```diff
@@ -8,3 +8,7 @@ plugins:
   - "@semantic-release/github"
 branches:
   - name: 'master'
+# maintaining v2
+  - name: 'release/2.x.x'
+    range: '2.x.x'
+    channel: '2.x.x'
```

Comment:
Tried to understand these properties by comparing them to the [official documentation](https://github.com/semantic-release/semantic-release/blob/master/docs/usage/workflow-configuration.md#branches-properties). Looks sane.